src/components/creator/CreatorInputs.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  FormGroup,
+  NumberInput,
+  Radio,
+  TextArea,
+  TextInput,
+} from '@patternfly/react-core';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+import React, { useMemo } from 'react';
+import SelectMultiTypeahead from '../SelectMultiTypeahead';
+import { ItemKind, itemKindMeta } from './meta';
+
+export type InputProps<T> = {
+  value: T;
+  onChange: (newValue: T) => void;
+};
+
+export const TypeInput = ({ value, onChange }: InputProps<ItemKind | null>) => {
+  const elementId = `rc-input-type`;
+
+  return (
+    <FormGroup label="Select content type" isRequired fieldId={elementId}>
+      <FormGroup
+        id={elementId}
+        role="radiogroup"
+        aria-label="Select content type"
+      >
+        {Object.entries(itemKindMeta).map(([rawName, meta]) => {
+          const name = rawName as keyof typeof itemKindMeta;
+
+          return (
+            <Radio
+              key={name}
+              id={`cr-input-type-${name}`}
+              name={`cr-input-type`}
+              isChecked={value === name}
+              onChange={(_, isChecked) => {
+                if (isChecked) {
+                  onChange(name);
+                } else if (value === name) {
+                  onChange(null);
+                }
+              }}
+              label={meta.displayName}
+            ></Radio>
+          );
+        })}
+      </FormGroup>
+    </FormGroup>
+  );
+};
+
+export const BundleInput = ({ value, onChange }: InputProps<string[]>) => {
+  const { getAvailableBundles } = useChrome();
+  const bundles = useMemo(() => getAvailableBundles(), []);
```

Comment:
No point for the memo.

Reply:
I did that out of a paranoid "what if the list of bundles changes" and not wanting to deal with handling that case. Sure, I'll remove that.